server/lib/notifications/agents/discord.ts

```diff
@@ -122,6 +122,7 @@ class DiscordAgent
         });
         break;
       case Notification.MEDIA_APPROVED:
+      case Notification.MEDIA_AUTO_APPROVED:
```

Comment:
This will cause a double notification? Right?

Reply:
Oh, I see. We will only send one type.

Reply:
Haha yeah, it's either one or the other.